funwithphysics/src/Components/Classical Mechanics/wpe_data.js

```diff
@@ -14,16 +14,17 @@ const Topics = [
     {
         topic: "Work",
         image: work,
-        details: `Work is said to be done when a body or object moves with the application of external force. We can define work as an activity involving a movement and force in the direction of the force. For example, a force of 20 newtons (N) pushing an object 3 meters in the same direction of the force will do 60 joules (J) of work.`,
+        details: `Work is said to be done when a body or object moves with the application of external force. We can define work as an activity involving a movement and force in the direction of the force. For example, a force of 20 newtons (N) pushing an object 3 meters in the same direction of the force will do 60 joules (J) of work .For more information visit ${<a href="https://en.wikipedia.org/wiki/Work_(physics)" target="_blank" rel="noreferrer">Wikipedia</a>}`,
         formula: "Work = Force × Displacement = F × d",
         process: "When we kick a football, we are exerting an external force called F and due to this force (kick), the ball moves to a certain distance. This disposition of ball from position A to B is known as displacement (d). This work is said to be done and can be calculated as W = F × d ",
         siunit: "Joule or Newton-Meter",
-        dimension: "ML²/T²"
+        dimension: "ML²/T²",
+        wikilink:"https://en.wikipedia.org/wiki/Work_(physics)"
```

Comment:
Why have u added wikilink separately??

Reply:
wait its old idea i will remove it

Reply:
@Padmanabh82 u haven't changed it why have u marked as resolved.

Reply:
done